runtime/server/controller.go

```diff
@@ -68,7 +69,8 @@ func (s *Server) ListResources(ctx context.Context, req *runtimev1.ListResources
 		if err != nil {
 			return nil, status.Error(codes.InvalidArgument, err.Error())
 		}
-		if !access {
+		// Check of the request is to skip security checks
+		if !req.SkipChecks && !access {
```

Comment:
Only admins should be allowed to pass `skip_checks=true` – otherwise anyone would be able to skip checks